chalk/core.py

```diff
@@ -111,6 +144,19 @@ def _repr_svg_(self) -> str:
         return svg
 
     def atop(self, other: "Diagram") -> "Diagram":
+        """Given to diagrams ``a`` and ``b``, ``a.atop(b)``
```

Comment:
"Given to" should be "Given t**w**o". This small typo also appears for other functions.

Reply:
Thanks. I had written for one and copy pasted for the others. 🤦‍♂️🤦‍♂️

Also, on a separate note, please check this out. 

- https://github.blog/2022-04-22-5-simple-things-every-developer-can-do-to-ship-more-secure-code/

Reply:
Made the corrections. Thank you for catching them.

![image](https://user-images.githubusercontent.com/10201242/172030645-d096d8c1-a1bd-4776-a40e-872cb7150bfa.png)